src/library/scala/collection/immutable/NumericRange.scala

```diff
@@ -250,10 +250,7 @@ extends AbstractSeq[T] with IndexedSeq[T] with Serializable {
       super.equals(other)
   }
 
-  override def toString() = {
-    val endStr = if (length > Range.MAX_PRINT) ", ... )" else ")"
-    take(Range.MAX_PRINT).mkString("NumericRange(", ", ", endStr)
-  }
+  override def toString = "%d %s %d by %s".format(start, if (isInclusive) "to" else "until", end, step)
```

Comment:
OK - I can change it to use the s-interpolator, but it does print BigDecimals wierdly:
```scala
println(Range.BigDecimal(new BigDecimal(1.3),new BigDecimal(100.3),new BigDecimal(1.3)))
```
prints out:
1.3000000000000000444089209850062616169452667236328125 until 100.2999999999999971578290569595992565155029296875 by 1.3000000000000000444089209850062616169452667236328125